### DISCUSSION TITLE
Kubernetes manifest files

### DISCUSSION THREAD

**Discussion by JPFrancoia** [ghostfolio/ghostfolio#2078] [General]
[manifests.zip](https://github.com/ghostfolio/ghostfolio/files/11778504/manifests.zip)

Someone asked for my manifest files to deploy ghostfolio on k8s, so here they are!

I'm using local storage to backup the DB. These files assume the presence of some secrets too, but I'll let you decide how you want to handle the secrets. Let me know if you have any question.

**Comment by adelowo** [comment]
Hi @JPFrancoia  , thanks for this. It would be great if you added a `secrets.yaml` file too. would be more one-click ish for people!.

Welldone on this!
